modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1491,8 +1491,8 @@ Output image must be of the same size and depth as input one.
 CV_32FC1, or CV_32FC2.
 @param map2 The second map of y values having the type CV_16UC1, CV_32FC1, or none (empty map
 if map1 is (x,y) points), respectively.
-@param interpolation Interpolation method (see cv::InterpolationFlags). The method INTER_AREA is
-not supported by this function.
+@param interpolation Interpolation method (see cv::InterpolationFlags). The methods INTER_AREA
+and #INTER_LINEAR_EXACT are not supported by this function.
```

Comment:
`INTER_LINEAR_EXACT`?

Reply:
Or `#INTER_AREA`

Reply:
Neither, but only INTER_AREA was in the docs.

Reply:
I mean `#` symbol.

Reply:
You got me. I made this first to imgproc.hpp and there it has an `#`. As I'm not really sure how the doc generation happens (and being not a C++ guy). I'm not sure. What do you think?

Reply:
Should be #INTER_AREA, yes. Can you add please?

check rendered documentation: https://pullrequest.opencv.org/buildbot/export/pr/18583/docs/d6/d91/group__gapi__transform.html#gac2c8de7ed8e75e46516246a8365b6298

Reply:
Did it in https://github.com/opencv/opencv/pull/18583/commits/09a62012b2e429d4e4ee8f27fdf5cdffd107c148 squashed and force-pushed at this branch. Not sure why is not reflecting here.